libs/utils/src/lib/models/operationmanifest.ts

```diff
@@ -193,7 +193,8 @@ export interface OperationManifestProperties {
   };
 
   customSwagger?: {
-    location: string[];
+    location?: string[];
+    operationId?: string;
```

Comment:
We can add here -
location?: ...
service: {
  name: ENUM [ApiManagement, AzureFunction]
  operationId: getSwaggerUrl [string;]
 }